cedar-policy-validator/src/typecheck/test/namespace.rs

```diff
@@ -140,51 +137,62 @@ fn namespaced_entity_get_attr() {
 #[test]
 fn namespaced_entity_can_type_error() {
     let src = r#"N::S::Foo::"alice" > 1"#;
-    assert_expr_typecheck_fails_namespace_schema(
+    let errors = assert_typecheck_fails(
+        namespaced_entity_type_schema(),
         Expr::from_str(src).expect("Expr should parse."),
         Some(Type::primitive_boolean()),
-        [ValidationError::expected_type(
+    );
+    let type_error = assert_exactly_one_error(errors);
+    assert_eq!(
+        type_error,
+        ValidationError::expected_type(
             get_loc(src, r#"N::S::Foo::"alice""#),
             expr_id_placeholder(),
             Type::primitive_long(),
             Type::named_entity_reference_from_str("N::S::Foo"),
             None,
-        )],
+        )
     );
 }
 
 #[test]
 fn namespaced_entity_wrong_namespace() {
-    assert_expr_typecheck_fails_namespace_schema(
+    let errors = assert_typecheck_fails(
+        namespaced_entity_type_schema(),
         Expr::from_str(r#"N::S::T::Foo::"alice""#).expect("Expr should parse."),
         None,
-        [],
     );
-    assert_expr_typecheck_fails_namespace_schema(
+    assert_eq!(errors.len(), 0);
```

Comment:
I'd rather `assert_eq!(errors, [])` so we get the actual contents of `errors` in any assertion failure.

Reply:
Good call

Reply:
Fixed in latest revision